### PR TITLE
Run PHPUnit test suite on builds.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
Run our PHPUnit test suite on all pull requests using Wercker. This is useful because now we can't accidentally merge code in that breaks the build. Automate all the things! :traffic_light: 

Like DoSomething/voting-app#435, this requires some modifications to the Wercker box that we're doing on every build, but we can clean that up later. In particular, it installs MongoDB and the PHP MongoDB extension.

#### How should this be manually tested?
The Wercker build [already did](https://app.wercker.com/#buildstep/563d24726709dc217b045c52)! ~~Well, almost – the web hook on the Wercker app for this repository broke when we changed repository names, and [needs to be fixed](https://github.com/DoSomething/northstar/issues/222).~~ And webhooks are now working again!

---
For review: @jonuy @angaither

### CODE REVIEW

File: wercker.yml [hunk 1]
```diff
@@ -4,15 +4,27 @@ build:
...
     - script:
-        name: update packages list
-        code: sudo apt-get update
-    - script:
         name: install php mongo extension
         code: sudo apt-get install php5-mongo
+    - script:
+        name: phpunit
+        code: |-
+          cp .env.example .env
```
Comment (jonuy) [inline wercker.yml]: Is there any concern about the .env.example file being modified in some way that it'll actually break tests when run on wercker?

Comment (DFurnes) [inline wercker.yml]: Ideally that'd always be a "clean slate" that you could just copy to get a new environment running. If it does break a build though, that's a good reminder for us to update instructions in the README or update the `.env.example`!

**Comment by DFurnes** [comment]
Wercker support [was able to update the repository name for our app](https://github.com/DoSomething/northstar/issues/222), and builds are now hooked up properly and showing up on pull requests again. :green_heart:

**Comment by jonuy** [comment]
:rocket:

**Pull request merged**